gui/src/elm/Icons.elm

```diff
@@ -1,11 +1,13 @@
 module Icons exposing (..)
 
-import Svg exposing (Svg)
-import Svg.Attributes
+import Html exposing (..)
+import Html.Attributes exposing (..)
+import Svg exposing (svg, node, path)
+import Svg.Attributes exposing (fill, d, viewBox)
 
 
 type alias IconFunction msg =
-    Int -> Bool -> Svg msg
+    Int -> Bool -> Svg.Svg msg
```

Comment:
Exposing the type is idiomatic:

```elm
import Svg exposing (Svg, svg, node, path)
...
type alias IconFunction msg =
    Int -> Bool -> Svg msg
```

The compiler will complain if you use a module instead of a type anyway.